File: blog-sever/server-java/src/main/java/com/site/blog/util/BeanMapUtil.java
package com.site.blog.util;

import com.site.blog.model.entity.BlogInfo;
import com.site.blog.model.entity.Comment;
import com.site.blog.model.vo.BlogDetailVO;
import com.site.blog.model.vo.CommentVo;
import org.springframework.beans.BeanUtils;


public class BeanMapUtil {
    public static BlogDetailVO copyBlog(BlogInfo blogInfo){
        BlogDetailVO blogDetailVO=new BlogDetailVO();
        BeanUtils.copyProperties(blogInfo,blogDetailVO);
        return  blogDetailVO;
    }
    public static CommentVo copyComment(Comment comment){
        CommentVo commentVo=new CommentVo();
        BeanUtils.copyProperties(comment,commentVo);
        return  commentVo;
    }
}

File: myblog-jpa/src/main/kotlin/com/site/blog/util/BeanMapUtil.kt
package com.site.blog.util

import com.site.blog.model.entity.BlogInfo
import com.site.blog.model.entity.Comment
import com.site.blog.model.vo.BlogDetailVO
import com.site.blog.model.vo.CommentVo
import org.springframework.beans.BeanUtils

object BeanMapUtil {
    @JvmStatic
    fun copyBlog(blogInfo: BlogInfo?): BlogDetailVO {
        val blogDetailVO = BlogDetailVO()
        BeanUtils.copyProperties(blogInfo!!, blogDetailVO)
        return blogDetailVO
    }

    @JvmStatic
    fun copyComment(comment: Comment?): CommentVo {
        val commentVo = CommentVo()
        BeanUtils.copyProperties(comment!!, commentVo)
        return commentVo
    }
}